flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/statemachine/StateMachineExample.java

```diff
@@ -105,7 +131,7 @@ public static void main(String[] args) throws Exception {
 				.flatMap(new StateMachineMapper());
 
 		// output the alerts to std-out
-		alerts.print();
+		alerts.writeAsText(outputFile, FileSystem.WriteMode.OVERWRITE);
```

Comment:
Do we really need to change this? `test_resume_savepoint.sh` does not change the example.